build.sh

```diff
@@ -16,16 +16,25 @@ if [ $# -lt 1 ]; then
   echo "- win32-arm64v8"
   echo "- linux-x64"
   echo "- linuxmusl-x64"
+  echo "- linuxmusl-arm64v8/arm64"
   echo "- linux-armv6"
   echo "- linux-armv7"
-  echo "- linux-arm64v8"
+  echo "- linux-arm64v8/arm64"
   echo "- darwin-x64"
   echo
   exit 1
 fi
 VERSION_VIPS="$1"
 PLATFORM="${2:-all}"
 
+# alias arm64 to arm64v8
```

Comment:
Do we still need these aliases? It looks like using `linuxmusl-arm64v8` as the `PLATFORM` might be sufficient.